### PR TITLE
Pre-release 0.43.1

### PR DESCRIPTION
**Pull request by jmurret**
Changes proposed in this PR:

- Update imageK8s to 0.43.1
- Update versions in Chart

How I've tested this PR:

:eyes:

How I expect reviewers to test this PR:

:eyes:

### CODE REVIEW

**Review by curtbushko — APPROVED** [review]
I compared this PR to the last release and nothing is jumping out at me.

https://github.com/hashicorp/consul-k8s/pull/1182/files

File: charts/consul/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: consul
-version: 0.43.0
+version: 0.43.1
```
Comment (ishustava) [inline charts/consul/Chart.yaml:3]: I think it should be 0.44 because there aren't just bug fixes in the changelog

Comment (jmurret) [reply charts/consul/Chart.yaml:3]: ooh!  thank you!

**Comment by jmurret** [comment]
opening a different branch and PR for 0.44.0.  Closing this.

**Pull request closed without merging**